truffle.js

```diff
@@ -5,6 +5,13 @@ module.exports = {
             port: 8545,
             network_id: "*" // Match any network id
         },
+        coverage: {
+            host: "localhost",
+            network_id: "*",
+            port: 8555,
+            gas: 0xfffffffffff,
+            gasPrice: 0x01
```

Comment:
BTW I was configuring gas limit on the TestRPC instance I was running. Maybe specifying this instead/in addition to that option would actually increase the gas limit? That way, we'd be able to run the FutarchyOracle tests.